### PR TITLE
TOMEE-3932 added technical document to help on the migration to Jakarta namespace

### PR DESCRIPTION
**Pull request by cesarhernandezgt**

### CODE REVIEW

File: namespace-change.adoc [hunk 1]
```diff
@@ -0,0 +1,246 @@
+= Migration to jakarta namespace
+
+== Introduction
+
+As described in the Apache TomEE Dev https://lists.apache.org/thread/qfxykymqbm916fhrpvhm8934vwo6pvt1[mailing list], the work for the migration to the Jakarta namespace is been track by https://issues.apache.org/jira/browse/TOMEE-3862[TOMEE-3862].
```
Comment (rzo1) [inline namespace-change.adoc:5]: is tracked by (?)

Comment (rzo1) [reply namespace-change.adoc:5]: dependency upgrades

File: namespace-change.adoc [hunk 2]
```diff
@@ -0,0 +1,246 @@
...
+The following is a list of common dependencies upgrades, configuration, tips and tricks that can help you to jump or speed up your contribution to this migration by fixing test, examples, etc. within the project.
+
+If you are new to the project, don't forget to check your https://tomee.apache.org/community/index.html[Community] page containing all the details on how to contribute.
```
Comment (rzo1) [inline namespace-change.adoc:8]: our

Comment (rzo1) [reply namespace-change.adoc:8]: all the details on how to start contributing ?

File: namespace-change.adoc [hunk 3]
```diff
@@ -0,0 +1,246 @@
...
+
+
+== General Notes
```
Comment (rzo1) [inline namespace-change.adoc:11]: Maybe we should say: Use a JDK >= 11 + Maven 3.8.x ?

File: namespace-change.adoc [hunk 4]
```diff
@@ -0,0 +1,246 @@
...
+----
+<dependency>
+    <groupId>org.apache.tomcat</groupId>
+    <artifactId>tomcat-api</artifactId>
+    <version>10.0.20</version>
+    <scope>provided</scope>
+</dependency>
+----
+b|
+Or latest Tomcat 10.0.x.
```
Comment (rzo1) [inline namespace-change.adoc:174]: "or latest Tomcat 10.0.x (depending on the Tomcat version used in the root pom.xml)" ?

File: namespace-change.adoc [hunk 5]
```diff
@@ -0,0 +1,246 @@
...
+----
+b|
+Shaded artifacts are described in the next section.
+
+
+|====
+
+
+=== Shaded artifacts
+Artifacts whose projects haven't release yet a jakarta namespace compatible version are being mmodified(shaded) in the module https://github.com/apache/tomee/tree/master/deps[tomee/deps], feel free to open a pull request if you find a version that needs to be shaded.
```
Comment (rzo1) [inline namespace-change.adoc:225]: Artifacts whose projects have not yet relased a ...

Comment (rzo1) [inline namespace-change.adoc:225]: mmodified -> modified

**Comment by rzo1** [comment]
Hi @cesarhernandezgt  - thanks! I added some wording-related comments.

**Comment by cesarhernandezgt** [comment]
Thank you @rzo1  for the review, I pushed the fixes in b520e3b0e2.

**Review by rzo1 — APPROVED** [review]

**Pull request merged**